CRM/Admin/Form/Setting/Mail.php

```diff
@@ -29,6 +29,7 @@ class CRM_Admin_Form_Setting_Mail extends CRM_Admin_Form_Setting {
     // dev/core#1768 Make this interval configurable.
     'civimail_sync_interval' => CRM_Core_BAO_Setting::MAILING_PREFERENCES_NAME,
     'replyTo' => CRM_Core_BAO_Setting::MAILING_PREFERENCES_NAME,
+    'civimail_unsubscribe_methods' => CRM_Core_BAO_Setting::MAILING_PREFERENCES_NAME,
```

Comment:
This is not the preferred way to add a setting to a form now - if you define the page in the setting metadata it should be added  - eg `    'settings_pages' => ['mail' => ['weight' => 90]],` (the rest have obviously not been converted)

Reply:
OK, so there exist two styles (`$_settings` vs `'settings_pages'`).  All the other elements of `civicrm/admin/mail` are defined in the style of `$_settings`. This is consistent. Using `settings_pages` for *this one setting* would make it a confusing outlier.

Of course, whenever the code style of `civicrm/admin/mail` is transitioned to `settings_pages`, this should go along with it.

Reply:
@totten we have been adding new settings in the preferred way to to other pages as we go - so the pattern of having a mix is not an outlier

Reply:
I put up a PR here to make it clear that the arrays declared are only a subset of the settings that may be on that page 
https://github.com/civicrm/civicrm-core/pull/28969